### ISSUE TITLE
Turning off Filterline requires refresh to take effect

### ISSUE THREAD

**Issue by fireattack** [honestbleeps/Reddit-Enhancement-Suite#3726]
- My browser (copy from https://www.whatismybrowser.com): Chrome
- My RES version: 5.2.0

As title said.

**Comment by andytuba** [comment]
Yes, this is a known issue for many aspects of RES. I'd like to sidestep this by adjusting user expectations: saving changes to options should show a notification that the page needs to be reloaded for changes to take affect.

**Comment by fireattack** [comment]
The problem (I found) is that, 

1. you don't need to refresh to turn it *on*
2. You don't need to refresh to turn it off for the previous "filter visited links" function, which is now integrated into this Filterline.
